Source/OxyPlot.Wpf/XpsExporter.cs

```diff
@@ -9,7 +9,6 @@
 
 namespace OxyPlot.Wpf
 {
-#if !NETFRAMEWORK
```

Comment:
cc @VisualMelon

I may be missing some context but this appears to work by fixing the condition in the csproj.

Reply:
I think we can scrub the condition in the csproj, but need to keep this, but I'm not familiar with this stuff.